cli/src/main/resources/hudson/cli/client/Messages.properties

```diff
@@ -2,11 +2,18 @@ CLI.Usage=Jenkins CLI\n\
   Usage: java -jar jenkins-cli.jar [-s URL] command [opts...] args...\n\
   Options:\n\
    -s URL       : the server URL (defaults to the JENKINS_URL env var)\n\
-   -i KEY       : SSH private key file used for authentication\n\
+   -http        : use a plain CLI protocol over HTTP(S) (the default; mutually exclusive with -ssh and -remoting)\n\
+   -ssh         : use SSH protocol (requires -user; SSH port must be open on server, and user must have registered a public key)\n\
+   -remoting    : use deprecated Remoting channel protocol (if enabled on server; for compatibility with legacy commands or command modes only)\n\
+   -i KEY       : SSH private key file used for authentication (for use with -ssh or -remoting)\n\
    -p HOST:PORT : HTTP proxy host and port for HTTPS proxy tunneling. See https://jenkins.io/redirect/cli-https-proxy-tunnel\n\
    -noCertificateCheck : bypass HTTPS certificate check entirely. Use with caution\n\
    -noKeyAuth   : don't try to load the SSH authentication private key. Conflicts with -i\n\
-   -v           : verbose output. Display logs on the console by setting j.u.l log level to FINEST\n\
+   -user        : specify user (for use with -ssh)\n\
+   -strictHostKey : request strict host key checking (for use with -ssh)\n\
+   -logger FINE : enable detailed logging from the client\n\
```

Comment:
Maybe. Is there any clear precedent? Winstone uses `--debug` with an argument, which is IMO more confusing.